distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc

```diff
@@ -3,58 +3,42 @@ The ${ddf-platform} application is considered to be a core application of the d
 The Platform application provides the fundamental building blocks that the distribution needs to run.
 These building blocks include subsets of:
 
-* http://karaf.apache.org/[Karaf], 
-* http://cxf.apache.org/CXF[CXF],  and 
-* http://camel.apache.org/[Camel]. 
+* http://karaf.apache.org/[Karaf] 
+* http://cxf.apache.org/CXF[CXF] 
+* http://camel.apache.org/[Camel] 
 
 A Command Scheduler is also included as part of the Platform application.
-The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals.
+The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals for the convenience of a "platform independent" method of running certain commands, such as backing up data or logging settings.
```

Comment:
this file is removed in my other pr. I'll edit it there and link to the changes from here.